cosmic-settings/src/main.rs

```diff
@@ -191,6 +217,127 @@ pub fn main() -> color_eyre::Result<()> {
+    let mode_str = if is_dark { "dark" } else { "light" };
+
+    if let Ok(mode_config) = ThemeMode::config() {
+        let mut mode = ThemeMode::get_entry(&mode_config).unwrap_or_default();
+        if mode.is_dark != is_dark {
+            let _ = mode.set_is_dark(&mode_config, is_dark);
+        }
+    }
+
+    let (theme_config, builder_config) = if is_dark {
+        (Theme::dark_config(), ThemeBuilder::dark_config())
+    } else {
+        (Theme::light_config(), ThemeBuilder::light_config())
+    };
+
+    if let Ok(config) = builder_config {
+        builder.write_entry(&config).map_err(|e| {
+            color_eyre::eyre::eyre!("Failed to write theme builder config: {:?}", e)
+        })?;
+    } else {
+        return Err(color_eyre::eyre::eyre!(
+            "Failed to get {} theme builder config",
+            mode_str
+        ));
+    }
+
+    let theme = builder.build();
+    if let Ok(config) = theme_config {
+        theme
+            .write_entry(&config)
+            .map_err(|e| color_eyre::eyre::eyre!("Failed to write theme config: {:?}", e))?;
+    } else {
+        return Err(color_eyre::eyre::eyre!(
+            "Failed to get {} theme config",
+            mode_str
+        ));
+    }
+
+    println!(
+        "Successfully imported {} theme from: {}",
+        mode_str,
+        path.display()
+    );
+
+    Ok(())
+}
+
+fn export_theme_cli(path: &std::path::Path) -> color_eyre::Result<()> {
+    use cosmic::cosmic_config::CosmicConfigEntry;
+    use cosmic::cosmic_theme::{ThemeBuilder, ThemeMode};
+
+    if path.extension().is_none_or(|ext| ext != "ron") {
+        return Err(color_eyre::eyre::eyre!(
+            "Theme file must have .ron extension: {}",
+            path.display()
+        ));
+    }
+
+    let mode_config = ThemeMode::config()
+        .map_err(|e| color_eyre::eyre::eyre!("Failed to get theme mode config: {:?}", e))?;
+
+    let mode = ThemeMode::get_entry(&mode_config).unwrap_or_default();
+    let is_dark = mode.is_dark;
+    let mode_str = if is_dark { "dark" } else { "light" };
+
+    let builder_config = if is_dark {
+        ThemeBuilder::dark_config()
+    } else {
+        ThemeBuilder::light_config()
+    }
+    .map_err(|e| {
+        color_eyre::eyre::eyre!("Failed to get {} theme builder config: {:?}", mode_str, e)
+    })?;
+
+    let builder = ThemeBuilder::get_entry(&builder_config).map_err(|e| {
+        color_eyre::eyre::eyre!("Failed to get {} theme builder: {:?}", mode_str, e)
+    })?;
+
+    let ron_string = ron::ser::to_string_pretty(&builder, Default::default())
+        .map_err(|e| color_eyre::eyre::eyre!("Failed to serialize theme to RON: {}", e))?;
+
+    std::fs::write(path, ron_string).map_err(|e| {
+        color_eyre::eyre::eyre!("Failed to write theme file '{}': {}", path.display(), e)
+    })?;
+
+    println!(
+        "Successfully exported {} theme to: {}",
+        mode_str,
+        path.display()
+    );
+
+    Ok(())
+}
+
```

Comment:
Can these two functions be moved to the the appearance module somewhere? Maybe in its own commands module.